src/main/fc/fc_tasks.c

```diff
@@ -410,6 +412,12 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .desiredPeriod = TASK_PERIOD_HZ(10),              // run every 100 ms, 10Hz
         .staticPriority = TASK_PRIORITY_HIGH,
     },
+    [TASK_SAVE_SETTINGS] = {
```

Comment:
This seems a bit like overkill for something that only happens once per flight at the point of disarm. Is there not a more efficient way of doing it using a flag set at Arming/Disarming that will achieve the same thing ?

Reply:
Sure. But I couldn't see anywhere relevant to system functions that is called at the start of the loop. Any suggestions are welcome. All that needs to happen is the 'processDelayedSave();' is called.

Reply:
I've removed the `SAVE_SETTINGS` task and put the call in to `SYSTEM`. This seems like the most suitable option of all the running tasks.